waku/node/peer_manager/peer_manager.nim

```diff
@@ -87,6 +89,8 @@ type PeerManager* = ref object of RootObj
   colocationLimit*: int
   started: bool
   shardedPeerManagement: bool # temp feature flag
+  topicsHealth*: Table[string, TopicHealth]
+  onTopicHealthChange*: TopicHealthChangeHandler
```

Comment:
Added these two parameters for feature-parity with go-waku's peer manager

https://github.com/waku-org/go-waku/blob/78b522db5063c6bb178dc5c16c41e1c308c4d7c0/waku/v2/peermanager/peer_manager.go#L70-L91

relay -> wakuRelay
subRelayTopics -> topicsHealth
TopicHealthNotifCh -> onTopicHealthChange

Reply:
I think `onTopicHealthChange` should also belong only to `WakuRelay`